src/docs/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import MintButton from 'components/mint-button';
+import DocsEntry from './docs-entry';
+import DocsBlock from './docs-block';
+
+export default (props) => {
```

Comment:
It's a new format they have in `0.14` release - component as a function. 
`import` is somehow not a dead code. If I remove it - the component breaks.
https://facebook.github.io/react/blog/2015/09/10/react-v0.14-rc1.html#stateless-function-components

It actually bothers me too. 
But I've decided to come back to `React.createClass` syntax, as it is mainstream right now and component as function does not play well with HMR.